content/applications/finance/accounting/get_started/shared_accounts.rst

```diff
@@ -0,0 +1,137 @@
+===============
+Shared Accounts
+===============
+
+In Odoo, accounts can be shared between multiple companies. This feature is particularly useful for
+businesses that operate under a **multi-company structure** and might benefit from a
+**unified chart of accounts**.
+
+Two ways are available to set up said feature: the :ref:`basic setup <setting_up_shared_accounts>`
+or using the :ref:`merge tool <merge_tool>`.
+
+The merging tool allows existing multi-company environments to use the new shared accounts feature.
+An :ref:`unmerge tool <unmerge_tool>` is also available to discontinue its use.
+
+This feature can also facilitate the :doc:`consolidation` of financial data across a multi-company
+environment, providing a **global view** of the financial situation for a group with child
+companies under a parent company.
+
+.. _shared_accounts_benefits:
+
+Benefits of Shared Accounts
+===========================
+
+1. Lighter load of accounts to handle
+-------------------------------------
+
+The main benefit of the shared accounts feature is to **reduce the amount of accounts** that have
+to be handled by the main accountant of a multi-company environment.
+
+.. example::
+   In environments with two companies using the same country’s generic chart of accounts, each
+   account used to appear twice. With the shared accounts feature, each account can now appear only
+   once.
+
+2. Simplified and Cleaner Reporting
+-----------------------------------
+
+The shared accounts feature **simplifies audit reports** while **keeping all data visible and
+accessible**, making the accounts easier to understand and manage.
+
+.. example::
+   In the balance sheet report, shared accounts between a company and its child company appear in a
+   single group, differentiated by the company using each account.
```

Comment:
I don't get what you mean by "differentiated by the company using each account" :thinking: